### ISSUE TITLE
An error will be reported when performing the re-scoring operation of n-gram --“Segmentation fault (core dumped)”

### ISSUE THREAD

**Issue by BADADH** [cffan/neural_seq_decoder#7]
Hello, after I downloaded the 5-gram Language model, I ran the corresponding eval_competition file and configured the lm_decoder parameter as follows:
nbest = lmDecoderUtils.lm_decode(
        ngramDecoder,
        logits[0],
        blankPenalty=blank_penalty,
        returnNBest=True,
        rescore=True,
    )
it will show the fault "Segmentation fault (core dumped)".  However it will operate successfully when rescore=False.  May I ask how this situation should be resolved. Thank you!
